app/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "MetaMask",
   "short_name": "Metamask",
-  "version": "4.0.12",
+  "version": "3.14.2",
```

Comment:
This will change the version on the uat branch. We will have to change it back to a `4.x.x` again when or before we merge to master, right?

Reply:
Woops, your correct. We will change it when uat -> master.